ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -52,6 +52,18 @@ IBTK_ENABLE_EXTRA_WARNINGS
         }                                                                                                              \
     } while (0);
 
+#ifdef IBTK_NDEF_IBTK_DEPRECATED
+#define IBTK_DEPRECATED(msg)
+#elif defined(__cplusplus) && (__cplusplus >= 201402L)
```

Comment:
It would probably be better if we detected whether the compiler supports `deprecated`...

Reply:
I can write a followup patch to add this to our feature detection.